vnext/local-cli/generator-windows/templates/cpp/src/MainPage.cpp

```diff
@@ -0,0 +1,46 @@
+﻿#include "pch.h"
+#include "MainPage.h"
+#include "MainPage.g.cpp"
+
+using namespace winrt;
+using namespace Windows::UI::Xaml;
+using namespace react::uwp;
+
+// clang-format off
```

Comment:
customer doesn't need this.

Reply:
This is used to get past an issue with yarn format mucking with the template variables.  I added logic to strip these out as the template is being copied (see vnext\generator-windows\index.js) so the developer won't see this when they run the CLI.

---
In reply to: [329280236](https://github.com/microsoft/react-native-windows/pull/3262#discussion_r329280236) [](ancestors = 329280236)